app/assets/stylesheets/views/_browse.scss

```diff
@@ -26,6 +26,17 @@ $browse-header-background-colour: #263135;
   overflow: auto;
 }
 
+.browse__breadcrumb-border {
+  background: govuk-colour("blue");;
+  height: 10px;
```

Comment:
My instinct with this kind of thing would be to use a border rather than setting the height and the background, but I think that's grounded in older browsers where regardless of content they'd often set a minimum height on elements based on the font size, so it's probably not a concern here. Interested in your thoughts though.

Reply:
@andysellick Thanks - I've taken all of this from existing CSS, so as to not rock the boat I didn't change it but happy to if you think it's worth it 👍